package.json

```diff
@@ -16,7 +16,7 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
   "lint-staged": {
-    "*.js": [
+    "{bin,config,scripts}/**/*.js": [
```

Comment:
I couldn't get the exclude to work, so I changed the pattern to match `format` and `format-check`.

Is that sufficient, or is it worth adding an extra glob to explicitly include `test/index.test.js`?